common/src/main/java/com/teamresourceful/resourcefullib/common/color/Color.java

```diff
@@ -102,16 +111,35 @@ public static Color createNamedColor(String name, int value) {
         return color;
     }
 
+    public static Color createPulsingColor(String name, int startingValue, Consumer<Color.ColorEditor> editorConsumer) {
+        Color color = new Color(startingValue);
+        color.specialName = name.toLowerCase(Locale.ENGLISH);
+        if (colorsWithNames.containsKey(color.specialName)) return colorsWithNames.get(color.specialName);
+        colorsWithNames.put(color.specialName, color);
+        Scheduling.schedule(() -> {
+            Color.ColorEditor editor = color.new ColorEditor();
+            editorConsumer.accept(editor);
+            color.updateValue();
+            color.updateFloats();
+        }, 0, 40, TimeUnit.MILLISECONDS);
+        return color;
+    }
+
     //endregion
 
     //region Parsers
 
     @Nullable
     public static Color tryParse(String color) {
-        if (color.startsWith("0x") || color.startsWith("#") || color.startsWith("0X"))
-            return new Color(Long.decode(color).intValue());
-        else if (colorsWithNames.containsKey(color.toLowerCase()))
+        if (color.startsWith("0x") || color.startsWith("#") || color.startsWith("0X")) {
```

Comment:
so we can clean up this logic a bit, can you do `color = color.toLowerCase(Locale.ROOT);` you can then remove the starts with "0X"